gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -617,6 +617,7 @@
   /** Specifies a static location in HDFS to upload jars to. Useful for sharing jars across different Gobblin runs.*/
   public static final String MR_JARS_DIR = "mr.jars.dir";
   public static final String MR_JOB_MAX_MAPPERS_KEY = "mr.job.max.mappers";
+  public static final String TARGET_MAPPER_SIZE = "target.mapper.size";
```

Comment:
Can you add the `mr` prefix to the key similar to the other configuration keys? e.g. `mr.target.mapper.size`